speed-dreams/source-2.2.3/src/modules/userinterface/legacymenu/mainscreens/optionsmenu.cpp

```diff
@@ -23,6 +23,8 @@
 #include <monitorconfig.h>
 #include <soundconfig.h>
 
+// SIMULATED DRIVING ASSISTANCE
+#include "ThresholdConfig.h"
```

Comment:
Isn't it more logical to add this entire menu to the dev screen instead of the options menu?

Reply:
The thresholds are for indication of interventions, so it changes how much you get spammed with intervention messages etc, so I imagine it is an option not only for the devs